docs/UserGuide.md

```diff
@@ -75,26 +75,46 @@ Shows a message explaning how to access the help page.
 Format: `help`
 
 
-### Adding a person: `add`
+### Adding a volunteer: `vadd` [COMING SOON]
 
-Adds a person to the address book.
+Adds a volunteer to an event.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
+Format: `vadd vid/VOLUNTEER_ID /to eid/EVENT_ID` or `vadd vn/VOLUNTEER_NAME /to en/EVENT_NAME`
 
-<box type="tip" seamless>
+Parameters:
+* vn/ - Volunteer name
+* vid/ - Volunteer id
+* eid/ - Event id
+* en/ - Event name
 
-**Tip:** A person can have any number of tags (including 0)
-</box>
+Restrictions:
+* The maximum number of characters of the event and volunteer is 50.
+* The event and volunteer name entered must exist.
+* The event id must be greater than or equal to 0 and lesser than the number of events existed.
+* The volunteer id must be greater than or equal to 0 and lesser than the number of volunteers existed.
 
 Examples:
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+* `vadd vid/1 /to eid/1`
+* `vadd n/Betsy Crowe /to en/fundraising`
+
+### Checking all volunteers : `vcheck` [COMING SOON]
```

Comment:
Likewise, "Checking all volunteers in event"

Reply:
yep